### PR TITLE
docs: fix script

### PR DESCRIPTION
**Pull request by Lightning00Blade**

### CODE REVIEW

**Review by sadym-chromium — APPROVED** [review]
this change looks like only renaming. Is it so?

**Comment by Lightning00Blade** [comment]
Its a rename but if fixes it as my previous change broke it.

**Review by OrKoN — APPROVED** [review]

File: tools/wpt-report-builder/formatter.mjs [hunk 1]
```diff
@@ -217,22 +217,21 @@ function generateHtml(map, commitHash) {
         })
 
         const expand = document.querySelector('.expand');
-        let toggleState = 'collapse';
+        let expandState = 'collapse';
```
Comment (OrKoN) [inline tools/wpt-report-builder/formatter.mjs:220]: any reason both of those states are not boolean?

Comment (Lightning00Blade) [reply tools/wpt-report-builder/formatter.mjs:220]: I use to change the name of button as well :sweat_smile:

**Pull request merged**